apps/README.md

```diff
@@ -189,6 +189,52 @@ fetching files served by the test runner, prefix the file path with
 document.write('<audio src="/base/test/audio/assets/win.mp3"/>');
 ```
 
+#### UI Component Style Guide ####
+
+We use `react-storybook` to generate a ui component style guide that you can use
+to discover what components are available to reuse as you build new
+features. You can also use the style guide to more easily develop new components
+without having to run all of code.org.
```

Comment:
I could use some more background info here. What is a storybook and when do I use it -- do I develop components within it? If I develop a component outside of the storybook, what do I have to do to make it show up in the storybook, or does it somehow get added automatically?